### ISSUE TITLE
GraphQL @function Not Being Called

### ISSUE THREAD

**Issue by Etep15** [aws-amplify/amplify-category-api#2107]
### Amplify CLI Version

12.8.2

### Question

I have the following schema.graphql...

```
type Task @model @auth(rules: [{ allow: private, provider: iam }]) {
  id: ID!
  requestUUID: String!
  accountUUID: String!
  status: Int!
}

type Mutation {
  addTask(
    id: ID!
    requestUUID: String!
    accountUUID: String!
    status: Int!
  ): Task 
  @function(name: "asyncTaskHandler-${env}")
}
```

I create a "Task" object from another lambda as described here in the docs...

[Connect to GraphQL API from AWS Lambda](https://docs.amplify.aws/javascript/build-a-backend/graphqlapi/connect-from-server-runtime/#connect-to-graphql-api-from-aws-lambda)

The Task shows up in DynamoDB. I've also turned on logging in AppSync and I see the log for "createTask". However I don't see anything for the "addTask". My function that is being called is very basic...

`exports.handler = async (event) => {
    console.log(`EVENT: ${JSON.stringify(event)}`);
    return event.arguments.msg;
};
`

At the very least I expect it some log output in the cloudwatch logs for that function. But there are no logs at all. 

I feel I'm missing a step?

**Comment by phani-srikar** [comment]
This seems to be related to https://github.com/aws-amplify/amplify-category-api/issues/2060. This regression is fixed in https://github.com/aws-amplify/amplify-category-api/pull/2062 and pending release.

**Comment by dpilch** [comment]
The fix is available in v12.10.0 of the CLI.
